tests/link_header_test.go

```diff
@@ -0,0 +1,63 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/xregistry/server/registry"
+)
+
+func TestLinkHeader(t *testing.T) {
```

Comment:
Let's move this into the http3_test file, at the bottom, and start its name with "TestHTTP" like the others in that file